### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Maxusha**
- new header (mobile, desktop, sub menu)
- 2 variants supported (dark, transparent)
- new combined logo of instamotion+allianz

### CODE REVIEW

File: packages/header/src/header.tsx [hunk 1]
```diff
@@ -1,51 +1,216 @@
...
+          <HelpWrapper>
+            <FormattedMessage id="header.menu.help" />
+          </HelpWrapper>
+          <CallerImg imgPath={imgPath} />
+          <Link
+            text={<FormattedMessage id="header.menu.phone_number" />}
+            color={textColor}
+            colorHover={textColorHover}
+            icon="phone"
+            path="tel:089-411151-100"
```
Comment (mrbabbs) [inline packages/header/src/header.tsx]: can we put the number as prop? we had a case that we had to change it :D on fly without deploy

File: packages/header/src/header.tsx [hunk 2]
```diff
@@ -1,51 +1,216 @@
...
+            colorHover={textColorHover}
+            icon="trophy"
+            path="/top-offers"
+          />
+          <Link
+            text={<FormattedMessage id="header.menu.wish_list" />}
+            color={textColor}
+            colorHover={textColorHover}
+            icon="star"
+            path="/wish-list"
```
Comment (mrbabbs) [inline packages/header/src/header.tsx]: we do not have `wish-list` url but `favoriten`

File: packages/header/src/header.tsx [hunk 3]
```diff
@@ -1,51 +1,216 @@
...
+            colorHover={textColorHover}
+            icon="search"
+            path="/autos"
+          />
+          <Link
+            text={<FormattedMessage id="header.menu.top_offers" />}
+            color={textColor}
+            colorHover={textColorHover}
+            icon="trophy"
+            path="/top-offers"
```
Comment (mrbabbs) [inline packages/header/src/header.tsx]: `top-offers` is `angebote`

File: packages/header/src/header.tsx [hunk 4]
```diff
@@ -1,51 +1,216 @@
...
+    height: 4rem;
+  }
+`;
 
-    &:hover {
-      cursor: pointer;
-    }
+const LogoWrapper = styled.div`
+  display: block;
+  width: 96px;
```
Comment (mrbabbs) [inline packages/header/src/header.tsx]: what about to use `rem`

File: packages/header/src/items/caller-img.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```
Comment (mrbabbs) [inline packages/header/src/items/caller-img.tsx]: we use camelCase for file naming

File: packages/header/src/items/mobile-menu.tsx [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+  flex-direction: column;
+  position: fixed;
+  top: 3rem;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  opacity: 1;
+  max-width: 425px;
```
Comment (mrbabbs) [inline packages/header/src/items/mobile-menu.tsx]: same here rem over px

File: README.md [hunk 7]
```diff
@@ -93,3 +93,27 @@ bolt version:bump
 
 It will ask you what package's version you want to bump and what would be the new version.
 It will bump the version of the package you've selected, and will find all the dependant packages and bump the version for them as well
+
```
[the remark this answers is not in the document]
Comment (Maxusha) [reply README.md]: I talked to Ivan, he said he deleted accidentally. I was in my branch, so I added it again.

File: packages/header/src/items/mobile-menu.tsx (hunk 6 above)
Comment (Maxusha) [reply packages/header/src/items/mobile-menu.tsx]: Fixed in 111e2ed

File: packages/header/src/header.tsx (hunk 3 above)
Comment (Maxusha) [reply packages/header/src/header.tsx]: Fixed in 111e2ed

File: packages/header/src/header.tsx (hunk 1 above)
Comment (Maxusha) [reply packages/header/src/header.tsx]: Fixed in 111e2ed

**Comment by Maxusha** [comment]
> Is it rebased with master if not please redo

It is rebased onto master.

**Pull request merged**